src/pool.c

```diff
@@ -94,6 +94,7 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #define MAX_DOWNSTREAM 8
 #define MAX_HOST 256
 #define MAX_RIG_ID 32
+#define MAX_RIGS 1024
```

Comment:
If `account_rl` took start and end buffer pointers, this isn't needed, as the function keeps adding while it finds workers and there is still space in the buffer.

Reply:
removed this definition.